lib/events/playback.go

```diff
@@ -107,6 +102,25 @@ func Export(ctx context.Context, rs io.ReadSeeker, w io.Writer, exportFormat str
 				if err != nil {
 					return trace.ConvertSystemError(err)
 				}
+			case teleport.YAML:
+				_, err := fmt.Println("---")
+				if err != nil {
+					return trace.ConvertSystemError(err)
+				}
+				err = utils.WriteYAML(w, event)
+				if err != nil {
```

Comment:
Nit: smallest scope principle.

```suggestion
				if err := utils.WriteYAML(w, event); err != nil {
```